boards/x86/qemu_x86/board.cmake

```diff
@@ -13,7 +13,7 @@ if(CONFIG_X86_64)
     # icount works with 1 CPU so we can enable it here.
     # FIXME: once this works across configs, remove this line and set
     # CONFIG_QEMU_ICOUNT_SHIFT in defconfig instead.
-    list(APPEND QEMU_EXTRA_FLAGS -icount shift=5,align=off,sleep=off -rtc clock=vm)
+    list(APPEND QEMU_EXTRA_FLAGS -icount shift=5,align=off,sleep=off -rtc clock=vm -machine q35)
```

Comment:
I think you want this outside of the if block. Or else Q35 is used when there is only 1 processor.

Reply:
ah, yes, will fix